spec/commands/edit_spec.rb

```diff
@@ -369,6 +369,18 @@ def last_exception; @pry.last_exception; end
         FileUtils.rm "spec/fixtures/foo.rb"
       end
     end
+
+    it "should write the evaluated command to history" do
+      quote = 'history repeats itself, first as tradegy...'
+      Pry.config.editor = lambda {|file, line|
+        File.open(file, 'w'){
+          |f| f << quote
```

Comment:
This is a bit weird, i've never seen the `|f|` on a line by itself before ;)

Reply:
Ugggh yea, it's _super_ weird. I must have had a nasty auto-formatter running on my editor.

Reply:
Let me send another PR fixing the formatting.